components/insight/SRC/org/openmicroscopy/shoola/env/data/OMEROGateway.java

```diff
@@ -479,6 +480,50 @@ private long getScriptID(SecurityContext ctx, String name, String message)
 		return -1;
 	}
 
+	/**
```

Comment:
Weird indent.

Reply:
insight indent is a disaster i.e. mix of tabs and spaces

Reply:
Happy for these minor issues *not* to be a blocker given that Insight is maintenance-only anyway.